File: JS/Problem 1: Multiples of 3 and 5.js
function multiplesOf3and5(number) {
  // Good luck!
  let sum = 0;
  for(let i = 1; i < number; i++){
    let mul3 = i % 3;
    let mul5 = i % 5;
    if(mul3 === 0 || mul5 === 0){
      sum += i;
    }
  }
  return sum;
}

console.log(multiplesOf3and5(1000))

File: Python/Problem 1: Multiples of 3 and 5.py
def multiplesOf3and5(num):
	sum = 0
	i = 1
	while i < num:
		mul3 = i % 3
		mul5 = i % 5
		if mul3 == 0 or mul5 == 0:
			sum += i
		i += 1
	return sum

print(multiplesOf3and5(1000))
